site/docs/v1/tech/guides/api-authorization.adoc

```diff
@@ -0,0 +1,116 @@
+* There is no browser interaction required in the credentials distribution process.
+
+Challenges of this approach:
+
+* There is no easy way to automatically expire the credentials. FusionAuth API keys are valid until deleted. And such deletion is a manual process.
+* Similarly, there is no programmatic way to create these keys. A human being must log in to the admin UI.
+* A FusionAuth API key is being used for business logic. This isn't really what these API keys are designed for.
+
+=== Authentication Tokens
+
+link:/docs/v1/tech/tutorials/application-authentication-tokens[Authentication tokens] and the link:/docs/v1/tech/apis/login[Login API] are the final option. 
+
+In this case, you'd create an application called 'Dev Center' and enable Authentication Tokens. The developers of the Reminder App would be associated with one or more users. When each user is created, an Authentication Token would also be created. Both the token and the user's email would be distributed to the Reminder App developers.
+
+Each time the Reminder App wanted to call the Todo API, it would present the email address and the authentication token. 
+
+The Todo API would in turn call the Login API against FusionAuth with these two credentials. FusionAuth would then either return a 200 if the program's user authenticated correctly and was registered with the 'Dev Center', a 202 if the application authenticated correctly but was not registered, or a 404 if the credentials were invalid. 
+
+Strengths of this approach:
+
+* There's no browser involved in acquiring credentials.
+* Generation of the email address and Authentication Token pair can be automated.
+* This is essentially a per user API key. This means you can use any of the user tools to manage them, such as locking a user to temporarily deny access.
+* Logins are logged and could be used for auditing purposes.
+
+Challenges of this approach:
+
+* Authentication Token generation and usage are not as secure as the OAuth grant. 
+* Authentication Tokens don't expire automatically.
+* You can't generate the credentials with the admin UI, only with the API.
+
+== Other considerations
+
+*Should you put the users of an API in a separate tenant?*
+
+The simplest choice is to create a separate application, such as the 'Dev Center' mentioned above. Tenants might be useful if you want multiple users with the same username or email to have different passwords, or if you need tenant scoped FusionAuth API keys to mitigate security risks.
+
+*Which is the best option?* 
+
+It all depends on your needs. Typically we recommend the OAuth option as that is the most secure, but recognize that having a user interact to get the initial JWT and refresh token may be problematic.
+
+*Can I rotate keys? That is, I'd like to have multiple credentials valid for one user so that I can distribute the newer credentials over time?*
+
+This is not currently possible, unfortunately.
```

Comment:
I would prefer not to say unfortunately since we own the product. :-)

Reply:
Fair!